generators/spa/templates/Vue3JS/common/packages/MSBuildTasks.1.4.0.88/tools/Build.proj

```diff
@@ -0,0 +1,65 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@alejoroman0605 Why isn't this using the same build files as the other module templates?